packages/ui-core/src/components/Pagination/components/PaginationNavigation/PaginationNavigation.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import cnCreate from 'utils/cnCreate';
+import PropTypes from 'prop-types';
+import './PaginationNavigation.less';
+import ArrowRightIcon from 'icons/System/24/Arrow_right_24.svg';
+import PaginationButton from 'components/Pagination/components/PaginationButton/PaginationButton';
```

Comment:
локальные зависимости лучше импортировать по относительному пути

```suggestion
import PaginationButton from '../PaginationButton/PaginationButton';
```